test/e2e/helpers/setupConnection.ts

```diff
@@ -54,4 +54,5 @@ export async function withConnection(invitatorUrl: string, receiverUrl: string)
     pin: receiverPin,
     stepCount: '2',
   })
+  await delay(2000)
```

Comment:
to update connection status on other node, mmm... yes, to be fair going to replace with something like polling and checking status on both sides like we have for integration test. Since I use node fetch I can not really use playwright's wait for, well I could but then in test suite we would need to add extra step to check for if pin has been verified

@mattdean-digicatapult what do you think?

Reply:
Just updated to await for correct status, e.g. 'Code Required' and also for 'Send Query' or return an error "timeout"